<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewIntersection"/>
   <addaction name="actionOpenIntersection"/>
   <addaction name="actionSaveIntersection"/>
   <addaction name="separator"/>
   <addaction name="actionEditIntersection"/>
  </widget>
  <action name="actionNewIntersection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbutton/icons/newDoc.svg</normaloff>:/toolbutton/icons/newDoc.svg</iconset>
   </property>
   <property name="text">
    <string>NewIntersection</string>
   </property>
  </action>
  <action name="actionOpenIntersection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbutton/icons/openDoc.svg</normaloff>:/toolbutton/icons/openDoc.svg</iconset>
   </property>
   <property name="text">
    <string>OpenIntersection</string>
   </property>
  </action>
  <action name="actionSaveIntersection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbutton/icons/saveDoc.svg</normaloff>:/toolbutton/icons/saveDoc.svg</iconset>
   </property>
   <property name="text">
    <string>SaveIntersection</string>
   </property>
  </action>
  <action name="actionEditIntersection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbutton/icons/Edit.svg</normaloff>:/toolbutton/icons/Edit.svg</iconset>
   </property>
   <property name="text">
    <string>EditIntersection</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
